platform/lib/platform_web/components.ex

```diff
@@ -3309,6 +3317,16 @@ defmodule PlatformWeb.Components do
                 <%= error_tag(@f, :location) %>
               </p>
             </div>
+            <div>
+              <%= label(@f, :address, "Address") %>
+              <%= text_input(@f, :address,
+                placeholder: "Enter the address (Main Street 123, City, Country)",
+                novalidate: true,
+                phx_debounce: "blur",
+                "x-on:input": "user_address = $event.target.value"
+              ) %>
+            </div>
+            <map-container-pin id="map-container-pin" lat={@lat} lon={@lon} />
```

Comment:
Is `map-container-pin` a custom component? If not, a `div` with an id `map-container-pin` might be more readable here.